.travis.yml

```diff
@@ -6,6 +6,7 @@ env:
   global:
     - ENCRYPTION_LABEL: "de8b5a0f10ef"
     - COMMIT_AUTHOR_EMAIL: "cam@mcc.id.au"
+    - secure: "IagnnmSy+JOF/Dqv25lSabCLeW5Rxh2VV6MJZjZuRhZR+CDTmH+ZdskHmk0TC19DH2bJ4t+LpBYrVjrCBu+6Ow923i3lNTQiVlXNIvTEZyWKcgttiTDjhytby6gE/CyuM1tPEC34IBnTUuWdURBHe7XFMCGQRDUD02troYxv4b1Jvv6+JzYtE90v6WVv0b0LjiNJ0ruftP7btjMypJoN40N5ObqNwIecL5RfJeEZoE01LtdrpxBEPSgPkv3VN+Fscjwpi8IASbeYLy294KDjRO1WOHx6i02DvJPnTLHdS0LUU6vBcLGRcz0+d760LPsExNRyJ4xpehewux0ueSXm2oMnujdWK0v9eUPrsoimzwL69c5yzCCAr+1PMVffOLAH8kcC2lklZ84LTvy3FGchh5SMfoXlgMwWoJSH4bub2yr38TB/skOem6OuO2M2PUdWkzAMRcj24LZhvCNSfkvy86q1ShtUFRG2mKLjmZa6FK0ONVUVeRbEjZ8PALM76g07c+uZcOAGu5A3xKoP2KOe0YR4rS4T++9oW8+svQDF4GiA5KqZiB8Yce6mW9NQ7/LQZpjllQtqiTATryJ3LW2yRzdo7k/Mz+ajHLhhiElKzTreRHTLZ/8YHvcL1wud1bDCAtoTOem/v3mxGHiXD6NvJZF6hz/Gf0guhN1yigHlNFI="
```

Comment:
I'm not too sure how this is supposed to work. Is this supposed to turn into `$TOKEN` later on? Or is there some other magic happening I'm not aware of here.